heartbeat/monitors/active/icmp/loop.go

```diff
@@ -344,7 +373,7 @@ func createListener(name, network string) *icmp.PacketConn {
 	//      true, even if error value itself is `nil`. Checking for conn suppresses
 	//      misleading log message.
 	if conn == nil && err != nil {
-		logp.Info("%v ICMP not supported: %v", name, err)
+		//logp.Info("%v ICMP not supported: %v", name, err)
```

Comment:
Leftover?

Reply:
bad habit on commenting instead of removing...  removing now...